otx/algorithms/classification/adapters/mmcls/models/classifiers/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""Classification classifiers."""
```

Comment:
Why import classifier and head files? Didn't we just talk about getting the backbone?

Reply:
it will be better to bring them too cause eventually we need to.. and they are all specifically implemented for cls. Do you think it's bad..?

Reply:
Do you have a plan about it? I thought we agree to bring them when I send the structure.

Reply:
This seems to need discussion.